### PR TITLE
Removes sleep delay and debug tools.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -44,72 +44,9 @@ public void init(){
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // System.out.println("Calls doGet"); DEBUG Tool
```
Comment (acmcarther) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Who LGTM'd this commented out code?

**Review by acmcarther — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (meagancheese) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Dunno, it's been there for a while though.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -119,8 +56,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
     final Gson gson = new Gson();
     jsonMessages = gson.toJson(messages);
```
Comment (GrantComm) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Could this be on one line? 
jsonMessages = new Gson().toJson(messages);

**Review by GrantComm — APPROVED** [review]
LGTM

**Pull request merged**